guides/common/modules/proc_deploying-the-ca-certificate-on-a-host-by-using-script-rex.adoc

```diff
@@ -0,0 +1,38 @@
+[id="deploying-the-ca-certificate-on-a-host-by-using-script-rex"]
+= Deploying the CA certificate on a host by using Script REX
+
+You can use remote execution (REX) with the Script provider to deploy the CA certificate.
+Redeploy the CA certificate when you change it on {ProjectServer}.
+
+.Prerequisites
```

Comment:
this block looks identical to guides/common/modules/proc_deploying-the-ca-certificate-on-a-host-by-using-ansible-rex.adoc -> I suggest to use a snippet for that.

If the job template is the only diff, then maybe you can use an attribute for the anchor, title, and job template and even reuse the whole file. This could also be a future improvement.

Reply:
I've snippetized the Prerequisites. The rest seems to have too many variables.